addons/account/models/product.py

```diff
@@ -139,6 +139,19 @@ def create(self, vals_list):
         products.invalidate_recordset(['taxes_id', 'supplier_taxes_id'])
         return products
 
+    def _get_gross_list_price(self, price):
+        self.ensure_one()
+        if self.taxes_id:
```

Comment:
if we do this: 
```
if not self.taxes_id:
    return price
```

we can gain one level of indentation so that the function is more readable